src/components/modal/index.tsx

```diff
@@ -71,7 +71,7 @@ const Modal: FC<PropsWithChildren<Props>> = ({
               justifyContent={
                 secondaryActionButton && primaryActionButton
                   ? 'space-between'
-                  : 'flex-end'
+                  : 'flex-start'
```

Comment:
did you check this change with all the UX-designers? this change will be in many places around the projects

Reply:
Do you know what is the reason behind this change?

Reply:
Yea, Waiting for confirmation on that one

Reply:
I have Green light!

Reply:
> Do you know what is the reason behind this change?

Oh, the reason is the latest design of saved searches modal which uses modal footers for action buttons.
https://www.figma.com/file/L4sLqmHGj2b08MUbWDA07g/Search-Results-%26-Advanced-Search-(Explorations)?node-id=4898%3A120426&mode=dev